src/styles/layout.scss

```diff
@@ -145,8 +145,8 @@ input {
   color: var(--color-text-primary);
 }
 
-body:not(.dark-mode) .dark-mode-only,
-.dark-mode .light-mode-only {
+body:not(.dark) .dark-mode-only,
```

Comment:
why change from `.dark-mode` to `.dark`?

Reply:
Because the plugin looks for the `.dark` css class

Reply:
[Here](https://www.gatsbyjs.com/plugins/@theowenyoung/gatsby-plugin-dark-mode/#implement-theming) is the link to the gatsby docs where they talk about it. Thanks for asking! It made me realize I need to change all the css files that refer to `.dark-mode`